src/urdf/urdf_parser/model.cpp

```diff
@@ -36,7 +36,7 @@
 #pragma warning(push, 0)
 
 #include <vector>
-#include "urdf_parser/urdf_parser.h"
+#include "../urdf_parser/urdf_parser.h"
```

Comment:
There are places (e.g., link.cpp) that have `#include "urdf_parser/urdf_parser.h"`. Do those need to change as well?

Reply:
Since it is a private copy and we don't expect anyone to use these headers outside of our code, probably we don't need to. The compilation is working since the way for [MSVC to include headers is curious](https://learn.microsoft.com/en-us/cpp/preprocessor/hash-include-directive-c-cpp?view=msvc-170):  for the quoted form, the second priority after the same directory of the header, is 

> 2) In the directories of the currently opened include files, in the reverse order in which they were opened. The search begins in the directory of the parent include file and continues upward through the directories of any grandparent include files.

So if I'm not wrong the first file in the mix when compiling the library is model.cpp and it opens the right directory. That is the reason why we don't need to change more. Might be a good idea anyway to change other occurrences just in case that we (or the compiler) modifies the order of object code.

Reply:
That is interesting and curious indeed. We should definitely add your comment in the file (maybe just model.cpp) for future readers.

Reply:
+1 to a comment about why this is necessary

Reply:
done ad05aa6d4d2f5c097b3ec2d7ce6e976ec1a68d82